bldnrnmacpkgcmake.sh

```diff
@@ -90,18 +98,18 @@ fi
 # from brew install gedit). User installations are expected to have the
 # former and would only accidentally have the latter.
 
-cmake .. -DCMAKE_INSTALL_PREFIX=$NRN_INSTALL \
+cmake .. -G Ninja -DCMAKE_INSTALL_PREFIX=$NRN_INSTALL \
   -DNRN_ENABLE_MPI_DYNAMIC=ON \
   -DPYTHON_EXECUTABLE=`which python3` -DNRN_ENABLE_PYTHON_DYNAMIC=ON \
   -DNRN_PYTHON_DYNAMIC="$pythons" \
   -DIV_ENABLE_X11_DYNAMIC=ON \
   -DNRN_ENABLE_CORENEURON=OFF \
-  -DNRN_RX3D_OPT_LEVEL=2 \
+  -DNRN_RX3D_OPT_LEVEL=1 \
```

Comment:
Will this make rxd slow for end users? If yes, as this is only at the time of creating installer, maybe it’s ok if this takes additional few minutes?

Reply:
I don't know. I turned it off since developing this PR requiired that I run the script quite a few times. I think I'll add an optional arg when I test run the script and the default will remain 2 for the final pkg build run.

Reply:
When this PR is merged into branch 8.2, running ```bash bldnrnmacpkgcmake.sh``` will use ```rxdoptlevel=2```.
It is just much faster to test the script using
```
rxdoptlevel=0 bash bldnrnmacpgkcmake.sh
```